homeassistant/components/cover/brunt.py

```diff
@@ -0,0 +1,168 @@
+"""
+Support for Brunt Blind Engine covers.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/cover/brunt
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.const import (
+    CONF_NAME, CONF_USERNAME, CONF_PASSWORD)
+from homeassistant.components.cover import (
+    CoverDevice, SUPPORT_OPEN, SUPPORT_CLOSE, SUPPORT_SET_POSITION,
+    ATTR_POSITION, PLATFORM_SCHEMA,
+    STATE_OPEN, STATE_CLOSED)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['brunt==0.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'brunt'
+ATTRIBUTION = 'Based on an unofficial Brunt SDK.'
+
+COVER_FEATURES = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+
+ATTR_COVER_STATE = 'cover_state'
+ATTR_CURRENT_POSITION = 'current_position'
```

Comment:
ATTR_CURRENT_POSITION is defined in home assistant.components.cover (just import)